doc/tutorial/02_layout/index.md

```diff
@@ -15,11 +15,12 @@ You can skip this chapter if you want to use raptor with the default IBF.
 # IBF vs HIBF
 
 Raptor works with the Interleaved Bloom Filter by default. A new feature is the Hierarchical Interleaved Bloom Filter
```

Comment:
that is is not true yet @eseiler is it? :D

We would need to tackle this soon then.

Reply:
How do you understand that sentence?
I kinda read it as: "IBF is the default", which is true. For HIBF, you need the layout, etc.

Reply:
Ups I misread it.